client/post-editor/post-editor.jsx

```diff
@@ -379,6 +379,11 @@ const PostEditor = React.createClass( {
 			this.setState( this.getInitialState(), function() {
 				this.refs.editor.setEditorContent( '' );
 			} );
+		} else if ( this.state.isNew && this.state.hasContent && ! this.state.isDirty ) {
+			// is a copy
```

Comment:
I think I've explained and justified that check to @aduth but definitely I forgot to write it in that comment too.